File: src/main/java/org/cimsbioko/activity/ManageFormsActivity.java
package org.cimsbioko.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import org.cimsbioko.R;

public class ManageFormsActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.manage_forms_activity);
        Toolbar toolbar = findViewById(R.id.manage_forms_toolbar);
        setSupportActionBar(toolbar);
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
            actionBar.setHomeButtonEnabled(true);
        }
        setTitle(R.string.manage_forms);
    }
}

File: src/main/kotlin/org/cimsbioko/activity/ManageFormsActivity.kt
package org.cimsbioko.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import org.cimsbioko.R
import org.cimsbioko.databinding.ManageFormsActivityBinding

class ManageFormsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTitle(R.string.manage_forms)
        ManageFormsActivityBinding.inflate(layoutInflater).apply {
            setContentView(root)
            setSupportActionBar(manageFormsToolbar)
        }
        supportActionBar?.apply {
            setDisplayHomeAsUpEnabled(true)
            setHomeButtonEnabled(true)
        }
    }
}